app/bundle/src/main/scala/org/bitcoins/bundle/gui/NeutrinoConfigPane.scala

```diff
@@ -46,6 +49,24 @@ class NeutrinoConfigPane(
     }
   }
 
+  private val networkList = new ObservableBuffer[String]()
+
+  private val networkComboBox: ComboBox[String] = new ComboBox[String]() {
+    networkList ++= BitcoinNetworks.knownNetworks.map(_.toString)
+    items = networkList
+
+    val networkName =
+      BitcoinNetworks.fromString(appConfig.chainConf.network.name).toString
+    selectionModel.value.select(networkName)
+    onAction = _ => {
+      if (peerAddressTF.text.value.contains(".suredbits.com")) { // Or should this use the peers.headOption logic?
```

Comment:
> // Or should this use the peers.headOption logic?

I don't think so, this looks correct to me

Reply:
However it may be worth adding this so when switching from regtest or signet you can get the suredbits nodes
```
|| peerAddressTF.text.value.contains("localhost") || peerAddressTF.text.value.contains("127.0.0.1")
```